ngraph/src/ngraph/coordinate_transform.hpp

```diff
@@ -24,9 +24,48 @@
 
 namespace ngraph
 {
-    class NGRAPH_API CoordinateTransform
+    class NGRAPH_API CoordinateIterator
```

Comment:
Please add documentation which explains when to use this class.

Reply:
fixed